docs/README.md

```diff
@@ -57,7 +57,7 @@
     * [render()](/docs/api/ShallowWrapper/render.md)
     * [setContext(context)](/docs/api/ShallowWrapper/setContext.md)
     * [setProps(nextProps)](/docs/api/ShallowWrapper/setProps.md)
-    * [setState(nextState)](/docs/api/ShallowWrapper/setState.md)
+    * [setState(nextState[, cb])](/docs/api/ShallowWrapper/setState.md)
```

Comment:
It would be nice to use `callback` for this variable (here and elsewhere), for maximum clarity.